v1.1/explore-the-admin-ui.md

```diff
@@ -0,0 +1,3 @@
+---
+redirect_to: admin-ui-overview.html
```

Comment:
It looks like we're adding this file back in just so we can put this front-matter field in place? If so, can you explain why we're doing this vs. using `redirect_from` in the other target file's front-matter? Also, can you add `redirect_to` to `CONTRIBUTING.md` as well? I'm confused as to how `redirect_from`, `redirect_to`, and `key` interact.